app/uk/gov/hmrc/gform/sharedmodel/formtemplate/ComponentType.scala

```diff
@@ -87,18 +89,20 @@ case class Address(international: Boolean) extends ComponentType with MultiField
 }
 
 case object Address {
-  val mandatoryFields = (id: FormComponentId) => List("street1").map(id.withSuffix)
-  val optionalFields = (id: FormComponentId) =>
+  val mandatoryFields: FormComponentId => List[FormComponentId] = (id: FormComponentId) =>
```

Comment:
Let's just say `id` instead of `(id: FormComponentId)` when we specify type explicitly.